apps/client/ui/Slider/Slider.tsx

```diff
@@ -1,66 +1,95 @@
+import { useState, useEffect, useRef } from 'react'
 import type { FC } from 'react'
 import 'rc-slider/assets/index.css';
-import { Range, createSliderWithTooltip } from 'rc-slider';
-
-//const { createSliderWithTooltip } = Slider;
-const SliderRange = createSliderWithTooltip(Range);
-//const { Handle } = Slider;
-
-/* const handle = props => {
-  const {
-    value,
-    dragging,
-    index,
-    ...restProps
-  } = props
-
-  return (
-    <SliderTooltip
-    prefixCls="rc-slider-tooltip"
-    overlay={`${value}`}
-    visible={true}
-    defaultVisible={true}
-    placement="top"
-    key={index}
-    align={{
-      offset: [0, -5],
-    }}
-    >
-      <>
-        <Handle value={ value } { ...restProps } >
-        </Handle>
-      </>
-    </SliderTooltip>
-  )
-} */
+import Slider from 'rc-slider';
 
 type CustomSliderProps = {
-  min?: number,
-  max?: number,
-  step?: number,
+  min?: number
+  max?: number
+  step?: number
   defaultValue?: number[]
-}
+  className?: string
+};
 
 export const CustomSlider: FC<CustomSliderProps> = ({
   min = 0,
   max = 10,
   step = 1,
   defaultValue = [0, 10],
+  className,
   ...props
-}) => (
-  <div className='min-w-[180px] m-5'>
-    <SliderRange
-      //handle={(props) => {return ( <span className={props.className} style={{ left: props.value+'%' }}> {props.value} </span> ) }}
-      min={min}
-      max={max}
-      step={step}
-      defaultValue={[min, max]}
-      tipProps={{ visible: true }}
-      trackStyle={[{ backgroundColor: 'var(--primary-color)', height: '4px' }]}
-      railStyle={{ backgroundColor: 'var(--dark-grey', height: '4px'}}
-      handleStyle={[{ backgroundColor: 'var(--primary-color)', height: '14px', width: '14px', border: 'none' }]}
-      className='w-full p-10'
-      {...props}
-    />
-  </div>
-);
+}) => {
+  const [values, setValues] = useState<number[]>(defaultValue);
```

Comment:
We don't want for our components to have their own state 😄 In most cases we just want from them to be a connector for the user with some other state in the app - in other words we want them to be controllers of outside state. To achieve this you need to create 'value` and `onChange` (or `handleChange` - depends on rules in the project) props to let component display and control outside value. You don't need to declare additional props if your generic props type contains them and they're passed to correct component (but it's not such case 😅 ) 👌